<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>add_drivers</class>
 <widget class="QWidget" name="add_drivers">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>746</width>
    <height>554</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton_back">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>86</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Назад</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_fio">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>441</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>281</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>ФИО (Пример: Иванов Иван Иванович)</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit_DateOfBirth">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>110</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Дата рождения</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Телефон (Пример:+7(777)777-77-77</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Номер водительского удостоверения</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2_DriverLicenseNumber">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Категория прав</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_DrivingCategory">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>380</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Дата выдачи прав</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit_LicenseStartDate">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>410</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>450</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Дата окончания прав</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit_LicenseFinishDate">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>480</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>510</y>
     <width>86</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Phone">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
